cpp/src/hash/fixed_hash_set.cuh

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <utilities/device_atomics.cuh>
+#include <cudf/detail/utilities/hash_functions.cuh>
+#include <hash/helper_functions.cuh>
+
+namespace cudf {
+
+/*
+ *  fixed set on a device
+ */
+template <typename Element, typename Hasher = default_hash<Element>,
+          typename Equality = equal_to<Element>>
+class device_fixed_hash_set {
```

Comment:
It'd be more consistent with the rest of the library to call this a device view.

Reply:
Done